osf/management/commands/move_egap_regs_to_provider.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import unicode_literals
+import logging
+
+from django.core.management.base import BaseCommand
+
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+from osf.models import (
+    RegistrationProvider,
+    RegistrationSchema,
+    Registration
+)
+from django.conf import settings
+
+def main(dry_run):
+    epag_provider = RegistrationProvider.objects.get(name=settings.EGAP_PROVIDER_NAME)
+
+    egap_schema = RegistrationSchema.objects.filter(
+        name='EGAP Registration'
+    ).order_by(
+        '-schema_version'
+    )[0]
+
+    egap_regs = Registration.objects.filter(registered_schema=egap_schema.id, provider___id='osf')
```

Comment:
Minor: `.exclude(provider___id='egap')` would technically be more correct, but as there aren't currently any non-`osf` `RegistrationProvider`'s, it's probably fine.